src/main/scala/io/confluent/examples/streams/algebird/CMSStoreChangeLogger.scala

```diff
@@ -44,7 +44,7 @@ class CMSStoreChangeLogger[K, V](val storeName: String,
     if (collector != null) {
       val keySerializer = serialization.keySerializer
       val valueSerializer = serialization.valueSerializer
-      collector.send(this.topic, key, value, this.partition, timestamp, keySerializer, valueSerializer)
+      collector.send(this.topic, key, value, null, this.partition, timestamp, keySerializer, valueSerializer)
```

Comment:
using `null` here as `context.headers()` results in an `IllegalStateException` as the `RecordContext` is null since the key and value is passed to the `RecordCollector` here on store flush.

Reply:
Don't understand the comment... If passing `null` result in `IllegalStateException` why is the change a proper fix?

Reply:
Sorry if my comments aren't clear. 
 
We need to pass `null` for headers as calling `context.headers()` is what causes the `IllegalStateException`.

Passing null for `headers` follows the pattern in  `org.apache.kafka.streams.state.internals.StoreChangeLogger`

Reply:
Ah. Makes sense know. Thanks for clarification.